types/office-js/index.d.ts

```diff
@@ -3351,16 +3346,16 @@ declare namespace Office {
          * 
          * - {@link https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets#textfile | TextFile} (when using `Office.FileType.Text`)
          *
-         * For add-ins running in Office host applications other than Office on iOS, the getFileAsync method supports getting files in slices of up 
-         * to 4194304 bytes (4 MB). For add-ins running in Office apps on iOS, the getFileAsync method supports getting files in slices of up to 
+         * For add-ins running in Office host applications other than Office on iOS, the `getFileAsync` method supports getting files in slices of up 
```

Comment:
For example, this talks about Office on iOS. I believe for WXP, it should be Office on iPad.

Reply:
Gotcha. Fixed.

Reply:
Changes look good. Thx.